### PR TITLE
build: input component

### PR DESCRIPTION
**Pull request by Yiwahpsp**
create input component

### CODE REVIEW

File: src/components/form/form-question.tsx [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import { FormColumn } from '@/types/form'
 import { UIDataType } from '@/types/ui-data'
 
+import { Input } from '../input'
```
Comment (shalluv) [inline src/components/form/form-question.tsx:4]: nit: personally, I prefer
```suggestion
import { Input } from '@/components/input'
```

File: src/components/input.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  onChange?: (event: React.ChangeEvent<HTMLInputElement>) => void
+  required?: boolean
+  pattern?: string
+  placeholder?: string
+  className?: string
+  disabled?: boolean
+  description?: string
+  error?: boolean
+  errorMessage?: string
+}) {
```
Comment (shalluv) [inline src/components/input.tsx:33]: Please extract this to `Interface InputProps = {}` or smth

**Review by shalluv — CHANGES_REQUESTED** [review]

**Comment by Yiwahpsp** [comment]
- change path input component
- create interface input props in file input.tsx

File: src/components/form/form-question.tsx [hunk 3]
```diff
@@ -1,5 +1,6 @@
 import { FormColumn } from '@/types/form'
 import { UIDataType } from '@/types/ui-data'
+import { Input } from '@components/input'
```
Comment (shalluv) [inline src/components/form/form-question.tsx:3]:
```suggestion
import { Input } from '@/components/input'
```

File: tsconfig.json [hunk 4]
```diff
@@ -19,7 +19,8 @@
     ],
     "baseUrl": ".",
     "paths": {
-      "@/*": ["./src/*"]
+      "@/*": ["./src/*"],
+      "@components/*": ["./src/components/*"]
```
Comment (shalluv) [inline tsconfig.json:23]: I think `@/components/` is fine. We don't need this

**Review by shalluv — CHANGES_REQUESTED** [review]
Also, please help provide me the image of the component. 
The screenshot of the example form page is fine.

**Comment by Yiwahpsp** [comment]
image

**Comment by Yiwahpsp** [comment]
image
ตอนโฟกัสคับบ

**Review by shalluv — APPROVED** [review]

**Pull request merged**